src/apps/Altinn.Authorization/src/Altinn.Authorization/Services/Implementation/ContextHandler.cs

```diff
@@ -411,6 +418,11 @@ protected async Task EnrichSubjectAttributes(XacmlContextRequest request, XacmlR
                     subjectOrgnNo = xacmlAttribute.AttributeValues.First().Value;
                 }
 
+                if (xacmlAttribute.AttributeId.OriginalString.Equals(XacmlRequestAttribute.PartyUuidAttribute))
+                {
+                    subjectPartyUuid = Guid.Parse(xacmlAttribute.AttributeValues.First().Value);
```

Comment:
Fristende å foreslå at vi burde ha sjekk her tilsvarende linje 473:
```
            if (!string.IsNullOrEmpty(subjectSsn) && subjectUserId != 0)
            {
                throw new ArgumentException("Not allowed to set userid and person-id for subject at the same time");
            }
```

På at man ikke oppgir både Fnr/UserId/UserUuid i samme request pga. mulighet for ambiguity. Men samtidig vet vi vel at standard bruk av dette fra PEP slenger på alle claims fra brukertoken.. så man får vel nå uansett både:
UserId, PartyId og PartyUuid på de fleste interne decision kall.

Reply:
Jepp. Det får være målet å få skrevet om dette i PEP på et eller annet tidspunkt til bare å sende den ene verdien